master/internal/experiment.go

```diff
@@ -606,6 +625,9 @@ func (e *experiment) setPriority(ctx *actor.Context, priority *int) error {
 		e.Config.SetResources(resources)
 		return errors.Wrapf(err, "setting experiment %d priority", e.ID)
 	}
+	if handler != nil {
```

Comment:
i also dont understand this. maybe it should be like `informRMOfChange` rather than a nil-able handler that it unused?